src/components/MetricsPane/MetricsPane.jsx

```diff
@@ -110,11 +120,10 @@ const MetricsPane = ({ showMetricsPane, setShowMetricsPane, view }) => {
   }, [displayedProjects, checkedProjects])
 
   const _setMetricsAfterCalculating = useEffect(() => {
-    const { numSurveys, numTransects, numUniqueCountries, yearRange } = calculateMetrics
+    const { numSurveys, numTransects, numUniqueCountries } = calculateMetrics
     setNumSurveys(numSurveys)
     setNumTransects(numTransects)
     setNumUniqueCountries(numUniqueCountries)
-    setYearRange(yearRange)
   }, [calculateMetrics])
```

Comment:
Continuing the pervious comment, this effect is unnecessary as the `calculateMetrics` object is already reactive.